tests/e2e/creation_test.go

```diff
@@ -68,6 +66,13 @@ func creationTestSuite(t *testing.T) {
 			err = testCtx.testDefaultCertsAvailable()
 			require.NoError(t, err, "error getting default cert secrets for Kserve")
 		})
+		t.Run("Validate all deployed components", func(t *testing.T) {
```

Comment:
It will be run 2 times, here and after Kserve. Intended?

Reply:
can you point to the line of the 2nd call? 
i cannot see it.....

Reply:
After this patch it looks like
```
		t.Run("Validate all deployed components", func(t *testing.T) {
			// this will take about 5-6 mins to complete
			err = testCtx.testAllApplicationCreation(t)
			require.NoError(t, err, "error testing deployments for DataScienceCluster: "+testCtx.testDsc.Name)
		})

		// Kserve
		t.Run("Validate Knative resoruce", func(t *testing.T) {
			err = testCtx.validateDSC()
			require.NoError(t, err, "error getting Knatvie resrouce as part of DataScienceCluster validation")
		})
		t.Run("Validate all deployed components", func(t *testing.T) {
			// this will take about 5-6mins to complete
			err = testCtx.testAllApplicationCreation(t)
			require.NoError(t, err, "error testing deployments for DataScienceCluster: "+testCtx.testDsc.Name)
		})
```
You remove it in later patch but it confuses this patch and spoils diff of the later patch.